### PR TITLE
Match Display

### PR DESCRIPTION
**Pull request by wayneholis**
Add image option for blockQuote (+1 squashed commit)
Squashed commits:
[62595c8] - fix `Link` regex
- remove `Linking` and replace to trigger via props, fix for relative URL
- `Strong` paragraph align to centre
- fix hash heading

### CODE REVIEW

File: rules.js [hunk 1]
```diff
@@ -13,11 +13,19 @@ var _ = require('lodash');
...
+  const LINK_HREF_AND_TITLE =
+          "\\s*<?([^\\s]*?)>?(?:\\s+['\"]([\\s\\S]*?)['\"])?\\s*";
   return {
     autolink: {
       react: function(node, output, state) {
         state.withinText = true;
-        var pressHandler = function() { Linking.openURL(node.target) };
+        var pressHandler = function() {
+          if(opts.onLink) {
+            opts.onLink(node.target)
```
Comment (frederickfogerty) [inline rules.js]: :+1:

File: rules.js [hunk 2]
```diff
@@ -189,10 +224,17 @@ module.exports = function(styles, opts={}) {
           }, output(node.content, state));
           state.withinParagraphWithImage = false
           return paragraph
+        } else if (_.size(node.content) < 3 && _.some(node.content, {type: 'strong'})){
+          // align to center for Strong
+          paragraphStyle = styles.paragraphCenter
+          console.log('here')
```
Comment (frederickfogerty) [inline rules.js]: :fire:

File: rules.js [hunk 3]
```diff
@@ -189,10 +224,17 @@ module.exports = function(styles, opts={}) {
           }, output(node.content, state));
           state.withinParagraphWithImage = false
           return paragraph
+        } else if (_.size(node.content) < 3 && _.some(node.content, {type: 'strong'})){
```
Comment (frederickfogerty) [inline rules.js:228]: `_.size(node.content) < 3` can you put a comment in the code for this?

File: rules.js (hunk 1 above)
Comment (frederickfogerty) [inline rules.js]: On second thoughts, can we refactor this to a top level function if it's repeated so many times?

File: rules.js (hunk 3 above)
Comment (wayneholis) [inline rules.js:228]: the `parse` is not working properly as it should, the whole paragraph of `strong` type will have a `text` object of empty string, yeah, i will put a comment on this

comments added

**Pull request merged**